MVP/src/index.ts

```diff
@@ -18,8 +18,10 @@ import { mapGQLResultToRepos } from './Processors/gqlProcessor';
 
 // temporary filepath
 const exampleFilepath = './src/Input/example_inFile.txt';
-const contents = ReadUrlFile(exampleFilepath);
-console.log(contents);
+let cleanUrls = InputHandling.ProvideURLsForQuerying(exampleFilepath);
+console.log(cleanUrls.github_URLs);
+console.log(cleanUrls.npm_URLs);
```

Comment:
![image](https://github.com/user-attachments/assets/96979b61-5b3b-4b82-bbc4-d35d678980a2)
Npm urls appear to be broken currently, showing package for packageName

Reply:
it's also worth mentioning that we should parse out '\n' and '\r'